src/content/reference/storage/_index.md

```diff
@@ -31,8 +31,4 @@ subscription.
 - [Use cases](/reference/storage/use-cases)
 - [Limits](/about/limits#kv)
 - [Pricing](/about/pricing#kv)
-- [Namespaces](/reference/storage/namespaces)
-- [Reading key-value pairs](/reference/storage/reading-key-value-pairs)
-- [Writing key-value pairs](/reference/storage/writing-key-value-pairs)
-- [Deleting key-value pairs](/reference/storage/deleting-key-value-pairs)
-- [Listing keys](/reference/storage/listing-keys)
```

Comment:
why is this all removed?

Reply:
These very small pages no longer exist and were combined into one page